molecule/config.py

```diff
@@ -39,19 +37,6 @@ def __init__(self, configs=[DEFAULT_CONFIG, PROJECT_CONFIG, LOCAL_CONFIG]):
     def molecule_file(self):
         return PROJECT_CONFIG
 
-    def populate_instance_names(self, platform):
```

Comment:
Was this relocated?  I don't see it moved elsewhere, however, we still have tests for this functionality.